docs/user/control-api.md

```diff
@@ -0,0 +1,189 @@
+# Controlling Glean
+
+Glean has a minimal control API available on its top-level `Glean` object.
+This API allows to enable and disable upload, register [custom pings][] and set [experiment data][experiments-api].
```

Comment:
This is missing the link for 'custom pings'.. but it seems to work anyway!? Interesting.

Reply:
Is it doing something smart like matching the name with the reference below?

Reply:
Good question, maybe? I would have imagined this to fail, given how `experiment data` is linked!

Reply:
It is! Short hand markdown form (I had that for the "experiments data" before and changed it, let me unify this here.

```suggestion
This API allows to enable and disable upload, register [custom pings][custom-pings] and set [experiment data][experiments-api].
```